### ISSUE TITLE
Could not pass event BlockPhysicsEvent to dynmap v3.2-beta-2-483

### ISSUE THREAD

**Issue by BlackAsLight** [webbukkit/dynmap#3418]
> template is **bold**
> sample data is *italicized*

**Issue Description:** *I get spammed with this error in the console every second some player is online.*

* **Dynmap Version:** *v3.2 beta 2 - spigot*
* **Server Version:** *paper 1.17 build 69*
* **Pastebin of Configuration.txt:** *https://pastebin.com/qd6DFzLY*
* **Pastebin of crashlogs or other relevant logs:** *https://pastebin.com/nvzV67An*
* **Steps to Replicate:** *Somebody logs onto my server and it starts spamming this until they log off.*

 [TICK] *I have looked at all other issues and this is not a duplicate*  
 [TICK] *I have been able to replicate this*

**Comment by bbayu123** [comment]
I would like to add to this issue since I'm getting the same error, but the replication is different.
This time, on **Paper 1.16.5 build 779**, all the errors show that it threw the stacktrace in BlockPhysicsEvents.
Stacktrace 1: https://pastebin.com/TEHdU1r5
Stacktrace 2: https://pastebin.com/UX8Lb1us

**Comment by wioxjk** [comment]
Getting this also but on 1.17.
The map works generally.

I do assume that a new .jar built for 1.17 will solve this.

**Comment by Compithegreat** [comment]
Got this as well in 1.17.1.

**Comment by FedUpWith-Tech** [comment]
Can replicate in Paper 1.17.1-91
With Dynmap 3.2-beta-2-483

**Comment by wioxjk** [comment]
Can confirm that this is fixed in the latest version that I pulled from github and built myself

**Comment by mikeprimm-triva** [comment]
Fix is in latest development builds - http://dynmap.us/builds/dynmap/Dynmap-3.2-SNAPSHOT-spigot.jar

**Comment by BrainStone** [comment]
Interesting. There have been no changes between beta-2 and the latest state on GitHub.  
Unless you were referring to my PR that is.

**Comment by RaidersPlay** [comment]
Tried the http://dynmap.us/builds/dynmap/Dynmap-3.2-SNAPSHOT-spigot.jar build
Seems like the exact same as the Dynmap v3.2-beta-2 that was on my server. Spammed messages still happening. 

I've also been having issues with server stability since 1.17.1 and the the start of the error spamming. When I disabled Dynmap there was an improvement. But we use Dynmap extensively on our server, so I enabled it again. The server HW is old, so might be more susceptible to instability. eg. CPU is a Intel i7-2630QM 2.9Ghz with 16GB mem and 125 GB SSD.
